vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -61,7 +61,23 @@ def process_collection_item(item, key)
 
     def parse_node(node)
       new_result = parse_base_item(node)
-      # TODO, add more properties such as resources
+
+      new_result.merge!(
+        :identity_infra   => node.spec.send(:externalID) || nil,
```

Comment:
Actually it seemed that calling send with a symbol was not failing if the method was missing. Since now we have kubeclient 0.1.10 anyway it can be simplified as you suggest (as missing info will return nil).